configs/CSC-Lumi-HPECray-zen3-MI250X-Slingshot/spack.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+spack:
+  packages:
+    default-compiler:
+      spack_spec: cce@16
+    default-mpi:
+      spack_spec: cray-mpich@8.1%cce ~gtl
```

Comment:
`gtl` isn't a variant on `cray-mpich`, and we don't have variants on compilers, so I think all this `+/-gtl` can be removed.

Reply:
I should point out that `LLNL-Tioga-HPECray-zen3-MI250X-Slingshot` also does this, and I think I've used that config to successfully generate experiments, so this might not be a breaking issue, but adding `~gtl` should not have any effect here, so keeping it would be misleading IMO.

Reply:
@scheibelp can you please make this change to the Tioga config, and run all the tests on Tioga with it for a sanity check?

If this indeed works on Tioga, please put up a PR to change this on Tioga AND the no-site version of the machine; I believe folks with similar hardware are starting with one of these two configs.

Reply:
I just realized that the benchpark Spack package repository defines a `cray-mpich` with a `+gtl` variant, so I was wrong about this.